packages/debugger/src/debugger.ts

```diff
@@ -120,7 +120,7 @@ export namespace Debugger {
 
     export const restartDebug = 'debugger:restart-debug';
 
-    export const pause = 'debugger:pause';
+    export const pauseOnExceptions = 'debugger:pause-on-exceptions';
```

Comment:
Final one (sorry for it), could you add a note about this api change in the migration guide:

https://github.com/jupyterlab/jupyterlab/blob/master/docs/source/extension/extension_migration.rst